examples/rfm7xAndBk242xCompatiblity/rfm7xAndBk242xCompatiblity.ino

```diff
@@ -0,0 +1,170 @@
+/*
+ * See documentation at https://nRF24.github.io/RF24
+ * See License information at root directory of this library
+ * Author: Brendan Doherty (2bndy5)
+ * Adapted for RFM75 by microtronics
+ */
+
+/**
+ * A simple example of sending data from 1 nRF24L01 transceiver to another.
+ *
+ * This example was written to be used on 2 devices acting as "nodes".
+ * Use the Serial Monitor to change each node's behavior.
+ */
+#include <SPI.h>
+#include "printf.h"
+#include "RF24.h"
+
+// See https://github.com/jnk0le/RFM7x-lib
+#include "rfm7x.h"
+
+// instantiate an object for the nRF24L01 transceiver
+RF24 radio(7, 8); // using pin 7 for the CE pin, and pin 8 for the CSN pin
+
+// Let these addresses be used for the pair
+uint8_t address[][6] = { "1Node", "2Node" };
+// It is very helpful to think of an address as a path instead of as
+// an identifying device destination
+
+// to use different addresses on a pair of radios, we need a variable to
+// uniquely identify which address this radio will use to transmit
+bool radioNumber = 1; // 0 uses address[0] to transmit, 1 uses address[1] to transmit
+
+// Used to control whether this node is sending or receiving
+bool role = false; // true = TX role, false = RX role
+
+// For this example, we'll be using a payload containing
+// a single float number that will be incremented
+// on every successful transmission
+float payload = 0.0;
+
+void setup()
+{
+
+  Serial.begin(115200);
+  while (!Serial) {
+    // some boards need to wait to ensure access to serial over USB
+  }
+
+  rfm7x_io_init();
+  spi_init();
+  if (rfm7x_is_present()) {
+    Serial.println("RFM7X connected, initializing");
+    //this code initializes the required registers for the clones. Especially bank1
+    rfm7x_init();
+    delay(2);
+    rfm7x_toggle_reg4();
+    delay(1);
+  }
```

Comment:
1. This code should reside in `RF24::begin()`. Since its use is conditional to the `RF24_FOR_RFM7X_BK242X` macro, the code should be encapsulated in
   ```cpp
   #if defined (RF24_FOR_RFM7X_BK242X)
       if (rfm7x_is_present()) {
           IF_SERIAL_DEBUG(printf_P(PSTR("RFM7X connected, initializing\r\n")));
           //this code initializes the required registers for the clones. Especially bank1
           rfm7x_init();
           delay(2);
           rfm7x_toggle_reg4();
           delay(1);
       }
   #else // !defined (RF24_FOR_RFM7X_BK242X)
   ```
2. `rfm7x_io_init()` and `spi_init()` should also be consolidated into `RF24::begin()`. Please try to find a way that these libraries could be better combined. Otherwise, I suggest you simply submit a link (that  we could add to the docs homepage) to a (your) personal repo that demonstrates using both libraries independently.